tool/tsh/common/tsh.go

```diff
@@ -1115,6 +1115,10 @@ func Run(ctx context.Context, args []string, opts ...CliOption) error {
 		}
 	}
 
+	// Remove HTTPS:// in proxy parameter as this is automatically handled
+	var httpsRE = regexp.MustCompile(`(?mi)https://`)
```

Comment:
Regex seems overkill here. Why not just a `TrimPrefix`?

Reply:
that would miss `HTTPS://` and just get `https://`. If we think that's a low enough scenario sure I can use TrimPrefix.

Reply:
@zmb3 converted to `TrimPrefix` of `https://` and `HTTPS://` which should cover most scenarios.